hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerDoubleBufferWithDummyResponse.java

```diff
@@ -163,14 +164,14 @@ private static class OMDummyCreateBucketResponse extends OMClientResponse {
     private final OmBucketInfo omBucketInfo;
 
     OMDummyCreateBucketResponse(OmBucketInfo omBucketInfo,
-        OMResponse omResponse) {
+                                OMResponse omResponse) {
       super(omResponse);
       this.omBucketInfo = omBucketInfo;
     }
 
     @Override
     public void addToDBBatch(OMMetadataManager omMetadataManager,
-        BatchOperation batchOperation) throws IOException {
+                             BatchOperation batchOperation) throws IOException {
```

Comment:
Minor nit: Here as well, we can stick with the original indent.

Reply:
I agree, keep original indent.

Also, please do not resolve without addressing the comment.